{"error":"[VertexAI.ClientError]: got status: 403 Forbidden. {\"error\":{\"code\":403,\"message\":\"Authentication error: 7; Error Details: User not allowed to access GCP services. Trace ID: 0x31f50270d5965af7\",\"status\":\"PERMISSION_DENIED\",\"details\":[{\"@type\":\"type.googleapis.com/google.rpc.ErrorInfo\",\"reason\":\"USER_BLOCKED_BY_ADMIN\",\"metadata\":{\"method\":\"google.cloud.aiplatform.v1.PredictionService.GenerateContent\",\"service\":\"aiplatform.googleapis.com\"}}]}}"}